terra/scripts/update_nanopore_tables.py

```diff
@@ -52,11 +48,11 @@ def load_summaries(gcs_buckets, project):
 
     ts = []
     for gcs_bucket in gcs_buckets:
-        blobs = storage_client.list_blobs(re.sub("^gs://", "", gcs_bucket))
+        blobs = storage_client.list_blobs(re.sub("^gs://", "", gcs_bucket), timeout=120)
 
         for blob in blobs:
             if 'final_summary' in blob.name:
-                doc = blob.download_as_string()
+                doc = blob.download_as_string(timeout=120)
```

Comment:
trivial (I can do it later in the overall refactoring): how about making this 120 a constant in this module?